packages/sdk/src/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './utils'
 export * from './cross-chain-messenger'
 export * from './adapters'
 export * from './l2-provider'
+export * from './contract-defs'
```

Comment:
Let's not export it for now and isntead plan on removing it completely in a future pr.  This abstraction is a bad abstraction for a bunch of reasons

- Hardcoded to only work on CJS doesn't work on any project using the more modern and recommended ESM
- Not typesafe
- Not code splittable, biggest tech debt the sdk has for frontend